src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_help.py

```diff
@@ -167,6 +167,11 @@
     short-summary: Lists the asset SAS URLs used for uploading and downloading asset content.
 """
 
+helps['ams asset get-encryption-key'] = """
+    type: command
+    short-summary: Gets the Asset storage encryption keys used to decrypt content created by version 2 of the Media Services API.
```

Comment:
It should be '**Get** the **a**sset storage..., see other commands for references.